pysat/instruments/jro_isr.py

```diff
@@ -193,17 +195,18 @@ def clean(self):
         if self.clean_level in ['clean', 'dusty', 'dirty']:
             print('WARNING: this level 2 data has no quality flags')
     else:
+        # Ion drift cleaning
         if self.clean_level in ['clean', 'dusty', 'dirty']:
             if self.clean_level in ['clean', 'dusty']:
                 print('WARNING: this level 2 data has no quality flags')
 
-            idx, = np.where((self['gdalt'] > 200.0))
+            ida, = np.where((self.data.indexes['gdalt'] > 200.0))
```

Comment:
in xarray if somethings is a coordinate you have to access it via .indexes?

Reply:
I need to get more up to speed on xarray :)

Reply:
Yup!  At least, that's the way it seems to be working.  Not an expert by any means.